activestorage/app/assets/javascripts/activestorage.js

```diff
@@ -550,7 +550,7 @@
       this.file = file;
       this.attributes = {
         filename: file.name,
-        content_type: file.type,
+        content_type: file.type || "application/octet-stream",
```

Comment:
Does it makes sense to set `null: false` for content_type in the migration, since it should never be NULL and causes errors if it is NULL?

Reply:
That sounds like a good idea. Would you like to make a PR?

Reply:
Done: #43055